plugins/module_utils/xfconf.py

```diff
@@ -1,16 +1,16 @@
 # -*- coding: utf-8 -*-
-# Copyright (c) 2022, Alexei Znamensky <russoz@gmail.com>
+# Copyright (c) 2024, Alexei Znamensky <russoz@gmail.com>
```

Comment:
Copyright lines should generally not be updated. I forgot the reasoning though (it was advice by some lawyers, if you ask on Matrix/IRC someone might remember)...

Reply:
```suggestion
# Copyright (c) 2022, Alexei Znamensky <russoz@gmail.com>
```